### PR TITLE
Added Varon Ratio function to image utils (unittests in tests_image_utils.py)

### PR DESCRIPTION
**Pull request by CalistaBesseling**
I'm not sure how much of a description you'd like, hopefully the title sums it up.

### CODE REVIEW

File: tests/test_image_utils.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+from src import image_utils
+import numpy as np
+
+""" Varon Ratio Tests """
```
Comment (pijner) [inline tests/test_image_utils.py]: Nice! Could you put this inside a class like what we have in [test_file_utils.py](https://github.com/WAT-ai/SatML/blob/main/tests/test_file_utils.py)?
Inheriting from `unittest.TestCase` will allow us to run multiple tests easily and not crash when one of the assertions fails

Comment (CalistaBesseling) [reply tests/test_image_utils.py]: Changes made 👍

File: tests/test_image_utils.py [hunk 2]
```diff
@@ -1,24 +1,27 @@
...
 import numpy as np
 
-""" Varon Ratio Tests """
-S1 = np.array([[0.1, 0.2, 0.3], [0.15, 0.25, 0.35], [0.2, 0.3, 0.4]])
-B1 = np.array([[1, 1, 1], [1, 1, 1], [1, 1, 1]])
-c = 10
-calculated_mean1 = 1.5
-calculated_stddev1 = 0.91287092917528
+class TestImageUtils(unittest.TestCase):
+    """ Varon Ratio Tests """
```
Comment (pijner) [inline tests/test_image_utils.py]: When I run `python -m unittest tests.test_image_utils`, I get "Ran 0 tests ..."

You probably want to put this in a class function called `test_varon_ratio(self)` and then replace the `assert` calls with `self.assertTrue()`.
For the `assert np.isclose()` calls, you can do `np.testing.assert_all_close()`, which takes in the same rtol value.

**Review by pijner — APPROVED** [review]
Looks great! Good to merge

**Pull request merged**